src/option/config/datazoom/index.js

```
@@ -11,43 +11,46 @@
-      if (style.handleStyle) {
-        dataZoom[0].handleStyle = Object.assign(dataZoom[0].handleStyle, style.handleStyle);
+  const self = iChartOption.dataZoom || {}
+  const dataZoom = toArray(self).map((item, index) => {
+    if (index === 0) {
+      const { show, position, start, end, startValue, endValue, style, height } = item;
+      const baseZoom = base()
+      if (show) {
+        end && (baseZoom.end = end);
+        start && (baseZoom.start = start);
+        if (startValue !== undefined) {
+          delete baseZoom.start;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)